File: core/src/SprCSharp/SprExport/tmp/structwrapper_PHHapticPointerDesc_HapticRenderMode.cpp
    __declspec(dllexport) int __cdecl Spr_PHHapticPointerDesc_struct_get_PHHapticPointerDesc_HapticRenderMode(HANDLE ptr, int index) {
        int _val = 0;
        try {
            PHHapticPointerDesc::HapticRenderMode* aryptr = (PHHapticPointerDesc::HapticRenderMode*) ptr;
            _val = (int) aryptr[index];
        }
        catch (SEH_Exception e) { e.raise_managed_exception("SprExport.dll"); }
        return _val;
    }
    __declspec(dllexport) HANDLE __cdecl Spr_PHHapticPointerDesc_struct_new_PHHapticPointerDesc_HapticRenderMode() {
        vector<PHHapticPointerDesc>* ptr = new vector<PHHapticPointerDesc>;
        return (HANDLE) ptr;
    }
    __declspec(dllexport) void __cdecl Spr_PHHapticPointerDesc_struct_set_PHHapticPointerDesc_HapticRenderMode(HANDLE ptr, int index, int value) {
        try {
            PHHapticPointerDesc::HapticRenderMode* aryptr = (PHHapticPointerDesc::HapticRenderMode*) ptr;
            aryptr[index] = (PHHapticPointerDesc::HapticRenderMode) value;
        }
        catch (SEH_Exception e) { e.raise_managed_exception("SprExport.dll"); }
    }

File: core/src/SprCSharp/SprCSharp/tmp/structwrapper_PHHapticPointerDesc_HapticRenderMode.cs
    public class structwrapper_PHHapticPointerDesc_HapticRenderMode : wrapper {
        private IntPtr _ptr;
        public structwrapper_PHHapticPointerDesc_HapticRenderMode(IntPtr ptr) { _ptr = ptr; }
        public structwrapper_PHHapticPointerDesc_HapticRenderMode(IntPtr ptr, bool flag) { _ptr = ptr; _flag = flag; }
        protected structwrapper_PHHapticPointerDesc_HapticRenderMode() {}
        ~structwrapper_PHHapticPointerDesc_HapticRenderMode() {}
        public PHHapticPointerDesc.HapticRenderMode this[int index] {
            get { return (PHHapticPointerDesc.HapticRenderMode) SprExport.Spr_PHHapticPointerDesc_struct_get_PHHapticPointerDesc_HapticRenderMode(_ptr, index); }
            set { SprExport.Spr_PHHapticPointerDesc_struct_set_PHHapticPointerDesc_HapticRenderMode(_ptr, index, (int) value); }
        }
    }
